.idea/codeStyles/Project.xml

```diff
@@ -7,7 +7,6 @@
       </value>
     </option>
     <option name="RIGHT_MARGIN" value="140" />
-    <option name="ENABLE_SECOND_REFORMAT" value="true" />
```

Comment:
This has been coming back since I switched to IJ 2023.2, LMK if this causes problems.